tests/unittests/test_salesforce_null_bookmark.py

```diff
@@ -0,0 +1,39 @@
+import logging
+from typing import Counter
+import unittest
+from unittest import mock
+# from tap_salesforce import main
```

Comment:
Remove unused/commented import statements.

Reply:
Remove and updated the code